src/utility/Display-ST7735/Display-ST7735.cpp

```diff
@@ -919,5 +919,8 @@ Rotation Display_ST7735::getRotation() {
 void Display_ST7735::invertDisplay(bool i) {
 	writecommand(i ? ST7735_INVON : ST7735_INVOFF);
 }
+displayPower(bool i) {
```

Comment:
Can you please add another blank line before it and explicitly say it is `void`? thank you!